cardano-cli/test/cardano-cli-golden/files/golden/help/latest_governance.cli

```diff
@@ -14,7 +13,6 @@ Available options:
 Available commands:
   create-mir-certificate   Create an MIR (Move Instantaneous Rewards)
                            certificate
-  action                   Governance action commands.
```

Comment:
This change shows the benefit of allowing commands and command groups to decide for themselves if they appear in an era.

All I did was make `governance action create-protocol-parameters-update` Conway onwards only.

Then the `action` command figured out that it was empty for all the pre-Conway eras and decided to not appear in those eras.